openam-core/src/main/java/com/sun/identity/idm/AMIdentityRepository.java

```diff
@@ -554,6 +554,27 @@ public void removeEventListener(int identifier) {
         }
```

Comment:
The copyright update is missing on this file

Reply:
Fixed